docs/source/en/model_doc/musicgen_melody.md

```diff
@@ -14,255 +14,170 @@ rendered properly in your Markdown viewer.
+inputs = processor(
+    text=["80s pop track with bassy drums and synth", "90s rock song with loud guitars and heavy drums"],
+    padding=True,
+    return_tensors="pt",
+)
+audio_values = model.generate(**inputs, do_sample=True, guidance_scale=3, max_new_tokens=256)
+```
 
+**Unconditional Generation**
 ```python
->>> from transformers import MusicgenMelodyForConditionalGeneration, MusicgenMelodyProcessor
-
->>> model = MusicgenMelodyForConditionalGeneration.from_pretrained("facebook/musicgen-melody")
->>> unconditional_inputs = MusicgenMelodyProcessor.from_pretrained("facebook/musicgen-melody").get_unconditional_inputs(num_samples=1)
+from transformers import MusicgenMelodyForConditionalGeneration, MusicgenMelodyProcessor
 
->>> audio_values = model.generate(**unconditional_inputs, do_sample=True, max_new_tokens=256)
+model = MusicgenMelodyForConditionalGeneration.from_pretrained("facebook/musicgen-melody")
+processor = MusicgenMelodyProcessor.from_pretrained("facebook/musicgen-melody")
+unconditional_inputs = processor.get_unconditional_inputs(num_samples=1)
+audio_values = model.generate(**unconditional_inputs, do_sample=True, max_new_tokens=256)
 ```
 
-### Generation Configuration
-
-The default parameters that control the generation process, such as sampling, guidance scale and number of generated tokens, can be found in the model's generation config, and updated as desired:
-
+**Generation Configuration**
+You can inspect and update the model's generation configuration.
 ```python
->>> from transformers import MusicgenMelodyForConditionalGeneration
-
->>> model = MusicgenMelodyForConditionalGeneration.from_pretrained("facebook/musicgen-melody")
-
->>> # inspect the default generation config
->>> model.generation_config
-
->>> # increase the guidance scale to 4.0
->>> model.generation_config.guidance_scale = 4.0
-
->>> # decrease the max length to 256 tokens
->>> model.generation_config.max_length = 256
+from transformers import MusicgenMelodyForConditionalGeneration
+
+model = MusicgenMelodyForConditionalGeneration.from_pretrained("facebook/musicgen-melody")
+# Inspect config
+print(model.generation_config)
+# Update config
+model.generation_config.guidance_scale = 4.0
+model.generation_config.max_length = 256
 ```
 
-Note that any arguments passed to the generate method will **supersede** those in the generation config, so setting `do_sample=False` in the call to generate will supersede the setting of `model.generation_config.do_sample` in the generation config.
+### Other Information
+- **Checkpoint Conversion**: Convert original checkpoints using the script at `src/transformers/models/musicgen_melody/convert_musicgen_melody_transformers.py`.
+- **`head_mask`**: The `head_mask` argument is only effective with `attn_implementation="eager"`.
+- **Sampling**: For best results, use sampling (`do_sample=True`).
 
 ## Model Structure
```

Comment:
Remove this section and replace with the below. Remember each code snippet should be indented under its list item

```suggestion
- [`MusicgenMelodyForCausalLM`] can be used as a standalone decoder model. Load it by specifying the correct config or accessing it through the `.decoder` attribute of [`MusicgenMelodyForConditionalGeneration`].

   [`MusicgenMelodyForConditionalGeneration`] can be used as a composite model that includes the text and audio encoder.
```